server/server_test.go

```diff
@@ -1322,12 +1324,28 @@ func startTestServer(config string) error {
 	if err != nil {
 		return err
 	}
+
 	go func() {
 		err := server.Start()
 		if err != nil {
 			return
 		}
 	}()
+
+	retry := 3
```

Comment:
I didn't notice it's a test suite (which is a composition of a separate tests) this constant may stay in this code. In the attached link, all similar usages are also only in tests.